tinyiiod.c

```diff
@@ -186,6 +186,34 @@ void tinyiiod_do_close(struct tinyiiod *iiod, const char *device)
 	tinyiiod_write_value(iiod, ret);
 }
 
+int32_t tinyiiod_do_writebuf(struct tinyiiod *iiod,
+			     const char *device, size_t bytes_count)
+{
+	int32_t ret;
+	size_t total_bytes = bytes_count;
+	char buf[256];
+	size_t offset = 0;
```

Comment:
I would declare the same type variables on one line and I would order the variable declarations from longest to shortest line.